site/docs/api/sources-sinks.md

```diff
@@ -950,7 +950,7 @@ p.readFrom(Sources.jdbc(
 
 The JDBC source only works in batching mode, meaning the query is only
 executed once, for streaming changes from the database you can follow the
-[Change Data Capture tutorial](../tutorials/cdc.md).
+[Change Data Capture tutorial](../tutorials/cdc-mysql.md).
```

Comment:
should be `cdc`

Reply:
Yeah, these are from parallel changes which I didn't notice. Will fix them all.